tutorial/stdlib.po

```diff
@@ -348,9 +348,8 @@ msgid ""
 "remote procedure calls into an almost trivial task.  Despite the modules "
 "names, no direct knowledge or handling of XML is needed."
 msgstr ""
-"使用 :mod:`xmlrpc.client` 和 :mod:`xmlrpc.server` 模組使實作遠端控制變得更為"
-"容易。使用前也不需要先了解相關知識或是掌握 XML 的技能，就能直接透過名稱使用模"
-"組。"
+"使用 :mod:`xmlrpc.client` 和 :mod:`xmlrpc.server` 模組使實作遠端程序呼叫變得"
```

Comment:
```suggestion
"使用 :mod:`xmlrpc.client` 和 :mod:`xmlrpc.server` 模組使實作遠端程式呼叫變得"
```

Reply:
I applied the suggestion here, but we could discuss the term. 😃 

This was my reference: https://terms.naer.edu.tw/
procedure: 程序,  program: 程式

Reply:
@StevenHsuYL 
I believe you're correct 👍 . I just found out that both Chinese terms are used across Internet but `遠端程序呼叫` is much more frequently-used than the another.
Sorry for the troublesome. You can change it back if you like 💯 .

Reply:
Thanks for reply.
In fact I'm not quite sure about my translation term, so the discussion like this will truly improve the quality and gain more consensus. 
I have changed it back. Thanks for nice review. 😃